dags/multipod/maxtext_end_to_end.py

```diff
@@ -103,15 +103,34 @@
               "cpu_device_type": CpuVersion.N2_STANDARD,
               "cpu_zone": Zone.US_CENTRAL1_B.value,
               "cluster_name": ClusterName.CPU_N2_STANDARD_64.value,
+              "time_out_in_min": 60,
           },
           {
               "script_name": "tpu/gemma/7b/2_test_gemma",
               "tpu_version": TpuVersion.V4,
               "tpu_cores": 16,
               "cluster_name": ClusterName.V4_16_MULTISLICE_CLUSTER.value,
               "tpu_zone": Zone.US_CENTRAL2_B.value,
+              "time_out_in_min": 60,
           },
-      ]
+      ],
+      "mixtral-8x7b": [
+          {
+              "script_name": "tpu/mixtral/8x7b/1_test_mixtral",
+              "cpu_device_type": CpuVersion.M1_MEGAMEM,
+              "cpu_zone": Zone.US_CENTRAL1_B.value,
+              "cluster_name": ClusterName.CPU_M1_MEGAMEM_96.value,
+              "time_out_in_min": 120,
```

Comment:
Just curious - is it taking more than 60mins for this script to run?

Reply:
Yes, that's right.